vignettes/O5_show_example.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "5. Include an example in the analysis paper outline"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{5. Include an example in the analysis paper outline}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction
+
+If you would like to see an example paper, that has code blocks to produce a summary table and a graphic, include the `example = TRUE` argument when you use the `make_project()` function.  For example, if you type `rUM::make_project("~/Desktop/my.example", "R", example=TRUE)`, in the **Results** section of the paper you will see code like:
+
+````r
```

Comment:
does the `r` need to be embraced like: `{r}`?

Reply:
No. That is just to control the color coding in Markdown (not R Markdown or quarto)